src/rgw/rgw_data_sync.cc

```diff
@@ -2530,11 +2534,12 @@ int RGWBucketShardIncrementalSyncCR::operate()
           marker_tracker.try_update_high_marker(cur_id, 0, entry->timestamp);
           continue;
         }
-        if (make_pair<>(entry->timestamp, entry->op) != squash_map[make_pair(entry->object, entry->instance)]) {
+        if (entry->instance.empty() &&
+            make_pair<>(entry->timestamp, entry->op) != squash_map[make_pair(entry->object, entry->instance)]) {
```

Comment:
can we revert the commit ce7d00ac1642d84c1d9111156a544c37801c5adf that added instance to the `squash_map` keys now that we're not using it?